src/utils/rocksdb_serialization.hpp

```diff
@@ -93,7 +95,7 @@ inline std::string SerializeVertexAsValueForAuxiliaryStorages(storage::LabelId l
   std::vector<storage::LabelId> labels_without_target;
   std::copy_if(vertex_labels.begin(), vertex_labels.end(), std::back_inserter(labels_without_target),
                [&label_to_remove](const auto &label) { return label_to_remove != label; });
```

Comment:
Isn't emplace_back with for loop will be better there also?

Reply:
storage::LabelId is trivially copyable so I don't think so but will change